roles/kubernetes/preinstall/templates/openstack-cloud-config.j2

```diff
@@ -3,4 +3,5 @@ auth-url={{ openstack_auth_url }}
 username={{ openstack_username }}
 password={{ openstack_password }}
 region={{ openstack_region }}
-tenant-id={{ openstack_tenant_id }}
\ No newline at end of file
+tenant-id={{ openstack_tenant_id }}
+domain-name={{ openstack_domain_name }}
```

Comment:
can you wrap it in a condition, so it doesn't break existing setups? 
{% if openstack_domain_name is defined and  not null %}